### PR TITLE
Fix crash in session detail screen

### LINKED ISSUE #196: [Assigned]Observed a crash. IllegalArgumentException in SpeakersSummaryLayout
## Overview (Required)

Observed a crash. 
Sorry I don't remember what operation I was performing, but pasted the relevant message from Logcat.

```
01-15 13:23:10.733 17822-17822/? E/AndroidRuntime: FATAL EXCEPTION: main
Process: io.github.droidkaigi.confsched2018.debug, PID: 17822
  java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter speakers
  at io.github.droidkaigi.confsched2018.presentation.common.view.SpeakersSummaryLayout.setSpeakers(Unknown Source:3)
  at io.github.droidkaigi.confsched2018.databinding.FragmentSessionDetailBinding.executeBindings(FragmentSessionDetailBinding.java:326)
  at android.databinding.ViewDataBinding.executeBindingsInternal(ViewDataBinding.java:379)
  at android.databinding.ViewDataBinding.executePendingBindings(ViewDataBinding.java:351)
  at android.databinding.ViewDataBinding$6.run(ViewDataBinding.java:178)
  at android.databinding.ViewDataBinding$5.onViewAttachedToWindow(ViewDataBinding.java:146)
  at android.view.View.dispatchAttachedToWindow(View.java:17388)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3319)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3326)
  at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1658)
  at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1386)
  at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6733)
  at android.view.Choreographer$CallbackRecord.run(Choreographer.java:911)
  at android.view.Choreographer.doCallbacks(Choreographer.java:723)
  at android.view.Choreographer.doFrame(Choreographer.java:658)
  at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:897)
  at android.os.Handler.handleCallback(Handler.java:789)
  at android.os.Handler.dispatchMessage(Handler.java:98)
  at android.os.Looper.loop(Looper.java:164)
  at android.app.ActivityThread.main(ActivityThread.java:6541)
  at java.lang.reflect.Method.invoke(Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```

### PR DESCRIPTION
**Pull request by androhi**
## Issue
- close #196 

## Overview (Required)
- Fixed the parameter to allow null.

## Reproduction procedure
1. Turn on `Don't keep activities` in developer options.
2. Start app and show session detail screen.
3. Tap home button and return home screen.
4. Tap recent app button and choose this app.

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/view/SpeakersSummaryLayout.kt [hunk 1]
```diff
@@ -86,12 +86,14 @@ class SpeakersSummaryLayout @JvmOverloads constructor(
      *     app:speakers="@{speakers}"
      *     />
      */
-    fun setSpeakers(speakers: List<Speaker>) {
-        speakerList.clear()
-        speakerList.addAll(speakers)
+    fun setSpeakers(speakers: List<Speaker>?) {
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/view/SpeakersSummaryLayout.kt]: 📝

**Comment by takahirom** [comment]
I learned. DataBinding property x keep activities Property will be null 👍
Thank you for your contribution! 👍

**Comment by androhi** [comment]
Thank you for quick response! I'm so happy!

**Pull request merged**